manifests/base/rbac/kustomization.yaml

```
@@ -1,4 +1,22 @@
+#
+# Copyright 2024 The Kubeflow authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 resources:
+  - serviceaccount.yaml
+  - clusterrole.yaml
+  - clusterrolebinding.yaml
```

We can't change this name since it is default name of `kubebuilder`
```suggestion
  - role.yaml
  - role_binding.yaml
```

---

Maybe we should manage the RBAC files manually in order to controller the RBAC rules with fine granularity.  I have updated the `make manifests` target to not generate RBAC files.

---

I am not sure that we should do that.
It is easier for us to leverage kubebuilder to define RBAC for each plugin that our runtime framework implements, so we always understand why we need to have this policy:
https://github.com/kubeflow/trainer/blob/master/pkg/runtime/framework/plugins/mpi/mpi.go#L61-L62



cc @tenzen-y